packages/app/src/specs/DirectoryItem.vue

```diff
@@ -3,11 +3,46 @@
     :title="props.name"
     class="flex text-sm py-4px items-center"
   >
-    <i-cy-chevron-down-small_x16
+    <svg
```

Comment:
Why does inlining these SVGs improve performance? This doesn't seem correct to me. The Vue compiler can tell when a component is static and should be able to make this optimisation for us under the hood.

Inlining these seems bad for maintainability - it's not clear where this icon came from, or how it was generated. 

As an aside, I have no idea how our icons work - where is `i-cy-chevron-down-small_x16` and how *is* it generated?

Also, we should be able to add a `v-once` to hint to Vue that this component is static. https://vuejs.org/api/built-in-directives.html#v-once. This might have the same impact as inlining the SVG.

Reply:
I know the SFC compiler quite well and I'd like to dig into this more. We hoist static components, so the icons we generate should be as optimised as an inline SVG under the hood. 

I'll need to learn a little more about our icon system. I think the real bug is in there, not in Vue - I have a feeling it's generating components that are not very optimised. 

Do you know much about our icons? Can we sync on this?